src/styles/main.scss

```
@@ -1,3 +1,88 @@
-body {
-  margin: 0;
+// Colors
+$gray: #eee;
+$pink: #ff69b4;
+$black: #000;
+
+// Sizes
+$cell-size: 100px;
+$border-size: 1px;
+$font-size: 30px;
+$padding-size: 10px;
+
+// Calculation variables
+$calendar-width: (7 * $cell-size) + (8 * $border-size) + (2 * $padding-size);
+$column-count: 7;
+
+// Mixins
+@mixin hover-styles {
+  cursor: pointer;
+  background-color: $pink;
+}
+
+// Modifier classes
+$start-days: (
+  "mon" 0,
+  "tue" 1 * ($cell-size + ($border-size * 2)) + 1px,
+  "wed" 2 * ($cell-size + $border-size * 2) + 2px,
+  "thu" 3 * ($cell-size + $border-size) + 6px,
+  "fri" 4 * ($cell-size + $border-size) + 8px,
+  "sat" 5 * ($cell-size + $border-size * 2) + 5px,
+  "sun" 6 * ($cell-size + $border-size * 2) + 5px
+);
```

1. move it to variables file
2. you shouldn't have calculation like this here, you can just write @each loop and use siple values like: 
```suggestion
$start-days: (
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6
);
```